src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/dibk/nabovarsel/models/default.metadata.json

```diff
@@ -0,0 +1,9097 @@
+{
+  "Org": "ttd",
```

Comment:
test runs green without this file. Delete?

Reply:
Agree. Need to add form layouts when we have to expand test at a later point